CHANGELOG.md

```diff
@@ -490,7 +510,8 @@ section                size        addr
 
 Initial release
 
-[Unreleased]: https://github.com/rust-embedded/cortex-m-rt/compare/v0.6.14...HEAD
+[Unreleased]: https://github.com/rust-embedded/cortex-m-rt/compare/v0.6.11...HEAD
```

Comment:
```suggestion
[Unreleased]: https://github.com/rust-embedded/cortex-m-rt/compare/v0.6.15...v0.6.x
```

Reply:
I had set it to 0.6.11...HEAD to show all the changes since then, but I think this is probably better for the 0.6 branch, thanks.